stubs/ansible/__init__.pyi

```diff
@@ -0,0 +1,3 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
It looks like `reuse lint` doesn't care about blank files (i.e. they don't need identifiers). I didn't check this before, hence I'm only pointing it out here :).

Reply:
I also noticed that, but I guess it doesn't hurt either in case someone ever wants to add code in here.